config.example.php

```diff
@@ -26,7 +26,7 @@
 HTML;
 
             $slidesLinkPattern = <<<'HTML'
-        <div itemprop="recordedIn" itemscope itemtype="http://schema.org/CreativeWork">
```

Comment:
Because google says `Erro: Page contains property "recordedin" which is not part of the schema.`